tests/components/google_assistant/test_trait.py

```diff
@@ -1865,7 +1865,10 @@ async def test_inputselector(hass):
     """Test input selector trait."""
     assert helpers.get_google_type(media_player.DOMAIN, None) is not None
     assert trait.InputSelectorTrait.supported(
-        media_player.DOMAIN, media_player.SUPPORT_SELECT_SOURCE, None, None
+        media_player.DOMAIN,
+        media_player.MediaPlayerEntityFeature.SELECT_SOURCE,
```

Comment:
Not related to this PR, would be nice to refactor this to import `MediaPlayerEntityFeature`

Reply:
Yeah agree :)